contributor_guidelines.md

```diff
@@ -0,0 +1,170 @@
+# Contributor Guidelines
```

Comment:
I would change this file name to CONTRIBUTING.md, I think it is more github idiomatic :)